build.gradle.kts

```diff
@@ -23,18 +23,18 @@ repositories {
 dependencies {
     // kotlin
     implementation(kotlin("stdlib"))
-    api("org.jetbrains.kotlinx:kotlinx-serialization-json:1.3.2")
+    api("org.jetbrains.kotlinx:kotlinx-serialization-json:1.3.3")
     // event system
     implementation("me.obsilabor:alert:1.0.5")
     include("me.obsilabor:alert:1.0.5")
     // fabric
-    minecraft("com.mojang:minecraft:1.19-pre1")
-    mappings("net.fabricmc:yarn:1.19-pre1+build.1")
-    modImplementation("net.fabricmc:fabric-loader:0.14.5")
+    minecraft("com.mojang:minecraft:1.19")
+    mappings("net.fabricmc:yarn:1.19+build.1")
+    modImplementation("net.fabricmc:fabric-loader:0.14.6")
```

Comment:
This just makes this mod more incompatible with older loader versions

Reply:
People who use an old fabric loader have only themselves to blame 🙃

Reply:
btw i will not retract the changes, either you accept the changes or close the pull request 🤔